include/z64save.h

```diff
@@ -142,8 +142,8 @@ typedef struct Save {
     /* 0x00F8 */ PermanentSceneFlags permanentSceneFlags[120];
     /* 0x0E18 */ u8 unk_E18[0x54];
     /* 0x0E6C */ u32 dekuPlaygroundHighScores[3];
-    /* 0x0E78 */ u32 pictoFlags0;
-    /* 0x0E7C */ u32 pictoFlags1;
+    /* 0x0E78 */ u32 pictoFlags0;                       // Flags set by `PictoActor`s if pictograph is valid
+    /* 0x0E7C */ u32 pictoFlags1;                       // Flags set by Snap_ValidatePictograph() to record errors; volatile since that function is run many times in succession
```

Comment:
```suggestion
    /* 0x0E78 */ u32 pictoSuccessFlags;                       // Flags set by `PictoActor`s if pictograph is valid
    /* 0x0E7C */ u32 pictoErrorFlags;                       // Flags set by Snap_ValidatePictograph() to record errors; volatile since that function is run many times in succession
```
I wonder if we could give these more interesting names now, and leave it to comments that the error flag uses `0x20` to indicate that it's an error (should that get a define?)

Reply:
I think the thing is that there isn't really any reason to use one over the other for each thing, it just happens to use the topmost few of the second one for error output.